androidApp/core/src/main/java/tech/takahana/iconwallpaper/android/core/ui/components/Button.kt

```diff
@@ -0,0 +1,43 @@
+package tech.takahana.iconwallpaper.android.core.ui.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun RoundButton(onClick: () -> Unit, backgroundColor: Color, text: String) {
+    TextButton(
+        onClick = onClick,
+        shape = CircleShape,
+        border = (BorderStroke(1.dp, Color(0xff707070))),
+        modifier = Modifier
```

Comment:
コンポーネントのようによく再利用されるものは、外からレイアウトできるように引数にModierを持っておきたいです。
```suggestion
fun RoundButton(modifier: Modifier = Modifier, onClick: () -> Unit, backgroundColor: Color, text: String) {
    TextButton(
        onClick = onClick,
        shape = CircleShape,
        border = (BorderStroke(1.dp, Color(0xff707070))),
        modifier = modifier
```